addon/-private/utils/parse-response-headers.ts

```diff
@@ -1,7 +1,11 @@
 const newline = /\r?\n/;
 
-export default function parseResponseHeaders(headersString) {
-  let headers = Object.create(null);
+export type ResponseHeaders =
+| {}
+| { [key: string]: string };
```

Comment:
Why have you chosen this type instead of ` { [key: string]: string } `?

Reply:
good question, I don't remember. everything seems fine without the extra `{}`